### PR TITLE
Translation updates

### PR DESCRIPTION
**Pull request by ruevs**
The new multi-constraints changed quite a few of the messages giving hints what the constraints can be applied to.
This updates the translations.

- _English_ is fine.
- _German_ and _Russian_: updated to the best of my abilities. @Evil-Spirit, @rurban, @carelinus can you please review them?
- _All other languages_: the changed messages will revert to English until updated. The old translated messages are saved at the bottom of the POT files. 
   @strzinek, @andesfreedesign @progval @TristeFigure @verylowfreq @mhalil @lomatus would you please take a look at your respective languages if you have time?

### CODE REVIEW

File: res/locales/fr_FR.po [hunk 1]
```diff
@@ -347,73 +347,48 @@ msgstr ""
...
+"    * a point and one to three plane faces (point on face(s))\n"
 msgstr ""
-"Mauvaise sélection pour la contrainte point / courbe / plan. Cette "
-"contrainte peut s'appliquer à :\n"
-"\n"
-"    – Deux points (points coïncidents)\n"
-"    – Un point et un plan de travail (point dans le plan)\n"
-"    – Un point et un segment de ligne (point en ligne)\n"
-"    – Un point et un cercle ou un arc (point sur courbe)\n"
-"    – Un point et une face plane (point sur une face)\n"
```
Comment (progval) [inline res/locales/fr_FR.po:368]: `Un point, et entre une et trois faces planes (point sur face(s))`

File: res/locales/fr_FR.po [hunk 2]
```diff
@@ -347,73 +347,48 @@ msgstr ""
...
-"    * four line segments or normals (equal angle between A,B and C,D)\n"
-"    * three line segments or normals (equal angle between A,B and B,C)\n"
-"    * two circles or arcs (equal radius)\n"
+"    * two or more circles or arcs (equal radius)\n"
 "    * a line segment and an arc (line segment length equals arc length)\n"
 msgstr ""
-"Mauvaise sélection pour une contrainte de longueur / rayon égale. Cette "
-"contrainte peut s'appliquer à :\n"
-"\n"
-"    – Deux segments de ligne (longueur égale)\n"
```
Comment (progval) [inline res/locales/fr_FR.po:388]: `Deux ou trois segments de ligne (longueur égale)`

File: res/locales/fr_FR.po [hunk 3]
```diff
@@ -463,37 +438,32 @@ msgstr ""
...
-"    * two points\n"
-"    * a line segment\n"
+"    * two or more points\n"
+"    * one or more line segments\n"
 msgstr ""
-"Mauvaise sélection pour la contrainte horizontale / verticale. Cette "
-"contrainte peut s'appliquer à :\n"
-"\n"
-"    – deux points\n"
-"    – Un segment de ligne\n"
```
Comment (progval) [inline res/locales/fr_FR.po:494]: `au moins deux points` and either:

* `au moins deux segments de ligne` (if they are always segments of the same line),
* `au moins deux segments de lignes` (if they have to be segments of different lines), or
* `au moins deux segments de ligne(s)` (if they can to be segments of different lines but don't have to be)

File: res/locales/fr_FR.po [hunk 4]
```diff
@@ -505,68 +475,55 @@ msgstr ""
...
 "    * a line segment and a normal\n"
 "    * two normals\n"
-msgstr ""
-"Mauvaise sélection pour une contrainte d'angle. Cette contrainte peut "
-"s'appliquer à :\n"
 "\n"
-"    – Deux segments de ligne\n"
-"    – Un segment de ligne et une normale\n"
-"    – Deux normales\n"
+"Eqaual angles:\n"
```
Comment (progval) [inline res/locales/fr_FR.po:495]: typo in English

File: res/locales/fr_FR.po [hunk 5]
```diff
@@ -505,68 +475,55 @@ msgstr ""
...
 "\n"
+"Angle between:\n"
 "    * two line segments\n"
 "    * a line segment and a normal\n"
 "    * two normals\n"
-msgstr ""
-"Mauvaise sélection pour une contrainte d'angle. Cette contrainte peut "
-"s'appliquer à :\n"
 "\n"
-"    – Deux segments de ligne\n"
```
Comment (progval) [inline res/locales/fr_FR.po:527]: Before this line, add `Angle entre :`. Make sure the whitespace before the colon is a non-breaking space like others in the file

File: res/locales/fr_FR.po [hunk 6]
```diff
@@ -505,68 +475,55 @@ msgstr ""
...
 "    * two line segments, arcs, or beziers, that share an endpoint (tangent)\n"
 msgstr ""
-"Mauvaise sélection pour la contrainte parallèle / tangente. Cette contrainte "
-"peut s'appliquer à :\n"
-"\n"
-"    – Deux segments de ligne (parallèles)\n"
-"    – Un segment de ligne et un parallèle (parallèle)\n"
-"    – Deux normales (parallèles)\n"
-"    – Deux segments de ligne, des arcs ou des Béziers, qui partagent une "
-"extrémité (tangente)\n"
```
Comment (progval) [inline res/locales/fr_FR.po:552]:
```
"    – Deux faces\n"
"    – Au moins deux segments de lignes (parallèle)\n"
"    – Au moins un segment de ligne et au moins une normale (parallèle)\n"
"    – Au moins deux normales (parallèle)\n"
"    – Deux segments de ligne, des arcs ou des Béziers, qui partagent une "
"extrémité (tangente)\n"
```

(this one had a couple typoes, I fixed them while I was at it)

File: res/locales/fr_FR.po [hunk 7]
```diff
@@ -1051,8 +1009,8 @@ msgid "&On Point / Curve / Plane"
 msgstr "&Sur Point / Courbe / Plan"
 
 #: graphicswin.cpp:157
-msgid "E&qual Length / Radius / Angle"
-msgstr "É&galité Longueur / Rayon / Angle"
+msgid "E&qual Length / Radius"
+msgstr ""
```
Comment (progval) [inline res/locales/fr_FR.po:1013]: `É&galité Longueur / Rayon`

**Review by progval — CHANGES_REQUESTED** [review]
Thanks for the ping. Github doesn't seem to let me suggest changes, so here are inline comments for French.

**Comment by Yprits** [comment]
I have some questions. Shouldn't lower case be used in any word except the first ones? E.g. "Открыть н&едавний" instead of "Открыть Н&едавний" and "Привязать выделение к &сетке" instead of "Привязать Выделение к &Сетке". 
And is there a way to _fully_ translate Property Browser?
image

**Comment by ruevs** [comment]
> And is there a way to _fully_ translate Property Browser?

Not at this point: https://github.com/solvespace/solvespace/issues/258#issuecomment-1175006630

As for the capital letters - Russian is not my native language - I would agree with you, but the Russian who translated it used capital letters.

**Review by strzinek — CHANGES_REQUESTED** [review]
Updated czech translation:
[cs_CZ.zip](https://github.com/solvespace/solvespace/files/10451191/cs_CZ.zip)

**Comment by Yprits** [comment]
> As for the capital letters - Russian is not my native language - I would agree with you, but the Russian who translated it used capital letters.

Every program with Russian language I know uses capital letters like this (as I said)
image

**Comment by mhalil** [comment]
Updated Turkish translation:
[tr_TR.po.zip](https://github.com/solvespace/solvespace/files/10460394/tr_TR.po.zip)

File: res/locales/fr_FR.po (hunk 2 above)
Comment (ruevs) [reply res/locales/fr_FR.po:388]: @progval This should be two or more, not two or three (as I think you translated it in French).
I'll use:
> au moins deux segments de ligne(s) (longueur égale)

borrowing from the next one. Let me know if it is not appropriate.

**Comment by rurban** [comment]
I get now just:
Missing (absent) translation for 'A&ngle / Equal Angle'
Missing (absent) translation for 'E&qual Length / Radius'

File: res/locales/fr_FR.po (hunk 4 above)
Comment (ruevs) [reply res/locales/fr_FR.po:495]: Will fix. Translated (with Google) as `Angles égaux :`

File: res/locales/fr_FR.po (hunk 3 above)
Comment (ruevs) [reply res/locales/fr_FR.po:494]: Used:
`Au moins un segment de ligne(s)`

**Comment by ruevs** [comment]
Hi all thanks for reacting.
I have force-pushed the branch with the updates so far..

### French:

@progval I was missing:
`two or more circles or arcs (equal radius)` (line 380) 
I used:
`Au moins deux cercles ou arcs (rayon égaux)`

Also see my comments above. I think I did the right thing but please double check. As well as all other changes.

### Turkish:

@mhalil your file restores the old Turkish messages, however there are changes in the English text that need to be translated.

### Czech:

Tanks @strzinek looks good.

**Comment by rurban** [comment]
I've recompiled it, with minor updates. see above.

**Comment by ruevs** [comment]
> I've recompiled it, with minor updates. see above.

@rurban as far as I can see only:

```
#: graphicswin.cpp:54
msgid "Im&port..."
msgstr "Im&portiere…" ```

I've merged it.

**Comment by mhalil** [comment]
I reviewed and edited the file again.
Can you check the file?
[tr_TR.zip](https://github.com/solvespace/solvespace/files/10469443/tr_TR.zip)

**Comment by mhalil** [comment]
Turkish Language File Updated

[tr_TR.zip](https://github.com/solvespace/solvespace/files/10475178/tr_TR.zip)

**Comment by ruevs** [comment]
> Turkish Language File Updated

@mhalil the zip file is still the old one, but I merged your updated version [from here](https://github.com/mhalil/solvespace/blob/master/res/locales/tr_TR.po) into this pull request. Please confirm that it is the right one.

**Comment by mhalil** [comment]
I fixed the file (I guess) :(
[tr_TR.zip](https://github.com/solvespace/solvespace/files/10502206/tr_TR.zip)

**Comment by ruevs** [comment]
Merged from your master.

**Comment by ruevs** [comment]
Updates remaining:
- Chinese - @lomatus @youzhou2020 
- Japanese - @verylowfreq
- Spanish - @andesfreedesign 
- Ukrainian - the translator is not involved any more, I could try to do it with a bit of help from Google :-)

**Comment by verylowfreq** [comment]
@ruevs Thanks for reminder!
Attached zip file contains the patch for ja_JP locale. (diff file and entire .po file)
[ja_JP_for_1331_230131.zip](https://github.com/solvespace/solvespace/files/10542542/ja_JP_for_1331_230131.zip)

**Comment by ruevs** [comment]
@verylowfreq thank you for the update, I merged it.

**Comment by ruevs** [comment]
Rebased and merged.

**Pull request closed without merging**